HISTORY.md

```diff
@@ -9,9 +9,12 @@ All notable changes to this project will be documented in this file.
     This is a follow-up of [#193].
 * Tests
   * Moved integration tests to a dedicated space and updated documentation for it. (via [#260]) 
+* Changed
+  * If 'homepage' property is solely a period, then omit 'website' entry from the externalReferences array. (via [#264])
```

Comment:
please also note the issue that is solved.  see an example below, where the issue is `#233` and the fix is `#247` 
```md
  * Private/protected properties of Component models are no longer directly accessible. ([#233] via [#247])  
```

Reply:
I have changed the changelog message to indicate the issue that the pull request fixes.